docs/index.md

```diff
@@ -89,3 +89,40 @@ def create_upload_file(file: UploadFile):
         session.commit()
         return {"filename": example.file.name}
 ```
+
+## Integration with Alembic
```

Comment:
I think this should be in the usage.md and only a sub-section of the sqlalchemy.